### PR TITLE
Feature - Update to use Dialog from ui-react

### PR DESCRIPTION
**Pull request by MoiyadKayda**
## Checklist:

- [x] I have updated the CHANGELOG with a summary of my changes

### CODE REVIEW

File: src/packages/cli/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "@kibalabs/core": "^0.5.3",
     "@kibalabs/core-react": "^0.6.0",
     "@kibalabs/everypage": "0.8.3",
-    "@kibalabs/ui-react": "^0.8.4",
+    "@kibalabs/ui-react": "file:/home/ubuntu/work/ui-react",
```
Comment (krishan711) [inline src/packages/cli/package.json]: undo

File: src/packages/cli/package.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
     "@kibalabs/core": "^0.5.3",
```
Comment (krishan711) [inline src/packages/cli/package.json:27]: i cant see if you have deleted the dialog and messagedialog from anvas. if you havent pleaase do.

File: src/packages/cli/package.json (hunk 1 above)
Comment (MoiyadKayda) [reply src/packages/cli/package.json]: ok

File: src/packages/cli/package.json (hunk 2 above)
Comment (MoiyadKayda) [reply src/packages/cli/package.json:27]: you mean the dialog files in everypage

Comment (krishan711) [reply src/packages/cli/package.json:27]: ye

**Review by krishan711 — APPROVED** [review]

**Pull request merged**